compose.yml

```diff
@@ -134,6 +134,28 @@ services:
       - .:/app
       - fileset-data:/data/filesets
 
+  packaging-listener:
```

Comment:
I'm thinking that the worker, listener, or consumer is per service/domain. It would listen to any exchanges/topics/queues for events or commands in its domain and takes whatever action.

To make it concrete, I think the ingest service is interested in packaging events and ingest events/commands.

Reply:
Okay, so I should combine `packaging-listener` with `ingest`? I can try that.

Reply:
Is this more what you had in mind? https://github.com/mlibrary/dor-py/pull/102/files/584759bc5c4c8d0323067376e46b9a9f21f0e5fc..a5cce0d17a86b385a2e8ab5daaff01ac8ad00c7c